fs_document/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2024 ERP|OPEN <https://www.erpopen.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+
+{
+    "name": "Base Document Attachment Object Store",
+    "summary": "Enable fetching external stored document.document",
+    "version": "16.0.1.0.0",
+    "author": "ERP|OPEN B.V. , Odoo Community Association (OCA)",
+    "license": "AGPL-3",
```

Comment:
This addon depends on an enterprise one. The `AGPL` licence is not compatible with the `OEEL-1` licence.

Reply:
@lmignon maybe i can add the if statement on the base module. Ir the document module is not installed it will still work